Add functions to run Suite2p processing (#8)
datajoint/element-interface: README.md

     loaded_dataset = suite2p_loader.Suite2p(output_dir)
     ```

> + Suite2p functions for processing

Reviewer: ```suggestion
+ Suite2p wrapper functions for triggering analysis
```

>      Each step of the Suite2p pipeline (registration, cell detection and 

Reviewer: ```suggestion
     Each step of the Suite2p pipeline (registration, cell segmentation and 
```

>      Each step of the Suite2p pipeline (registration, cell detection and 
>      deconvolution) can be run independently. The functions will facilitate this
>      process. The user will need to define the ops and the db dictionary before
>      one or all of the processing steps of Suite2p are run.

Reviewer: ```suggestion
     + Each step of Suite2p (registration, segmentation and deconvolution) 
     can be run independently. The functions in this package will facilitate 
     this process. Requirements include the [ops dictionary](
     https://suite2p.readthedocs.io/en/latest/settings.html) and db dictionary.
```

> 

Reviewer: ```suggestion
+ These wrapper functions were developed primarily because `run_s2p` cannot individually run deconvolution using the `spikedetect` flag ([Suite2p Issue #718](https://github.com/MouseLand/suite2p/issues/718)).
```

     ```python

>      from Suite2p_trigger import motion_correction_suite2p, segmentation_suite2p

Reviewer: ```suggestion
     from element_data_loader.suite2p_trigger import motion_correction_suite2p, segmentation_suite2p, deconvolution_suite2p
```

>      from variables import ops, db

Reviewer: ```suggestion
```

     from suite2p import default_ops

     ops = dict(default_ops(), nonrigid=True, two_step_registration=False)

     db = {
          'h5py': [], # a single h5 file path
          'h5py_key': 'data',
          'look_one_level_down': False, # whether to look in ALL subfolders when 
                                        # searching for tiffs
          'data_path': ['/test_data'], # a list of folders with tiffs 
                                   # (or folder of folders with tiffs if 
                                   # look_one_level_down is True, or subfolders 
                                   # is not empty)                                    
          'subfolders': [], # choose subfolders of 'data_path' to look in (optional)
          'fast-disk': '/test_data' # string which specifies where the binary file 
                                   # will be stored (should be an SSD)
          }
     

>      reg_ops = motion_registration_suite2p(ops, db)

Reviewer: ```suggestion
     registration_ops = motion_registration_suite2p(ops, db)
```

>      roi_ops = segmentation_suite2p(reg_ops, db)

Reviewer: ```suggestion
     segmentation_ops = segmentation_suite2p(registration_ops, db)
```

>      roi_ops, spikes = deconvolution_suite2p(roi_ops, db)

Reviewer: ```suggestion
     deconvolution_ops, spikes = deconvolution_suite2p(segmentation_ops, db)
```

     ([Suite2p Issue #718](https://github.com/MouseLand/suite2p/issues/718)).

     ```python
     from element_data_loader.suite2p_trigger import motion_correction_suite2p,
     segmentation_suite2p, deconvolution_suite2p
     from suite2p import default_ops

     ops = dict(default_ops(), nonrigid=True, two_step_registration=False)

     db = {
          'h5py': [], # a single h5 file path
          'h5py_key': 'data',
          'look_one_level_down': False, # whether to look in ALL subfolders when 
                                        # searching for tiffs
          'data_path': ['/test_data'], # a list of folders with tiffs 
                                   # (or folder of folders with tiffs if 
                                   # look_one_level_down is True, or subfolders 
                                   # is not empty)                                    
          'subfolders': [], # choose subfolders of 'data_path' to look in (optional)
          'fast-disk': '/test_data' # string which specifies where the binary file 
                                   # will be stored (should be an SSD)
          }
     

>      registration_ops = motion_registration_suite2p(ops, db)
>      segmentation_ops = segmentation_suite2p(registration_ops, db)
>      deconvolution_ops, spikes = deconvolution_suite2p(segmentation_ops, db)

Reviewer: ```suggestion
     ops.update(do_registration = 1, roidetect = False, spikedetect = False)
     registration_ops = motion_registration_suite2p(ops, db)
     
     registration_ops.update(do_registration = 0,  roidetect = True, spikedetect = False)
     segmentation_ops = segmentation_suite2p(registration_ops, db)
     
     segmentation_ops.update(do_registration = 0, roidetect = False, spikedetect = True)
     deconvolution_ops, spikes = deconvolution_suite2p(segmentation_ops, db)
```
